server/src/main/java/no/nav/foreldrepenger/vtp/server/api/journalforing/hendelse/JournalforingHendelseSender.java

```diff
@@ -47,16 +46,15 @@ private GenericData.Record lagGeneriskJournalfoeringHendelseRecord(JournalpostMo
 
     private String tilHendelsesType(JournalpostModell modell) {
         return switch (modell.getJournalStatus() != null ? modell.getJournalStatus().getKode() : "") {
-            case "J" -> "Journalført";
-            case "MO" -> "Mottatt";
-            case "M" -> "MidlertidigJournalført";
-            case "A" -> "Avbrutt";
-            default -> "MidlertidigJournalført";
+            case "J" -> "EndeligJournalført";
```

Comment:
Disse hendelsestypene stemte ikke helt med det som står dokumentert, og ble dermed sortert ut i k9-Fordel: https://confluence.adeo.no/display/BOA/Joarkhendelser#

Jeg vet ikke helt hvordan dette vil slå ut på den andre testene, men prøver denne kombinasjonen for nå. Tar gjerne imot forslag til andre konfigurasjoner.